.github/workflows/canary.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Use Node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
```

Comment:
the main branch doesn't use `actions/setup-node` anymore and is currently using the guild shared CI workflows which should handle that there. so I'm gonna close this